### PR TITLE
matplotlib changes

### PR DESCRIPTION
**Pull request by ezapartas**
quick solution to #318 issue of the newest version of matplotlib

### CODE REVIEW

**Comment by ezapartas** [comment]
@ka-rocha consider whether we need to make a similar change in active_learning/psy_cris, where `get_cmap` is also found in the code.

**Comment by tassos25** [comment]
We should first make a decision about which matplotlib version we want to use, and if it is 3.9, we should specify it in the requirements files. Only then this PR should be accepted.

**Comment by sgossage** [comment]
The easy route is likely downgrade our new v2 environments to matplotlib 3.7.1 which we know works, otherwise we should probably run tests to ensure 3.9.0 does not break our code in any additional unexpected ways

**Comment by mkruckow** [comment]
We made a downgrade of the required matplotlib in preparation for the f2f-meeting, see #336 . Hence, please merge the current development into your branch and free the installation restriction again within this PR.

**Comment by astroJeff** [comment]
Update (July 25): Manos to include updates to setup.py (or equivalent) the installed/required version of matplotlib. Can we also check if this code is backwards compatible with older matplotlib versions?

File: posydon/visualization/plot_pop.py [hunk 1]
```diff
@@ -2,10 +2,10 @@
     "Simone Bavera <Simone.Bavera@unige.ch>",
     "Max Briel <max.briel@unige.ch>",
     ]
-    
+
 import os
 import numpy as np
-import matplotlib.pyplot as plt
```
Comment (mkruckow) [inline posydon/visualization/plot_pop.py:8]: This can't be removed, `plt` is still used at other places.

Comment (ezapartas) [reply posydon/visualization/plot_pop.py:8]: It exists again a few lines below, it was duplicate there, probably by an earlier error

Comment (mkruckow) [reply posydon/visualization/plot_pop.py:8]: I'd probably remove the other instance, to keep the matplotlib imports together.

**Comment by astroJeff** [comment]
Update (Aug 1): @ezapartas to meet with @dimsour94 to update tomorrow - hopefully accept in the next couple days.

**Comment by mkruckow** [comment]
I made a test with this branch to create some plots with the pipeline on the CO-HMS grid: all fine!

**Review by mkruckow — APPROVED** [review]
I made a test with this branch to create some plots with the pipeline on the CO-HMS grid: all fine!

**Review by dimsour94 — APPROVED** [review]

**Pull request merged**